ZUGFeRD/AssociatedDocument.cs

```diff
@@ -39,6 +39,13 @@ public class AssociatedDocument
         /// </summary>
         public string LineID { get; internal set; }
 
+        /// <summary>
+        /// refers to a the superior line
```

Comment:
As this is a very special case, can you add a sentence of how to use this? Would it make sense, to even add a code sample in the inline documentation?